frontend/util.js

```diff
@@ -54,6 +63,22 @@ export const isInvalidStudentId = (number) => {
     return !number.match(/^[0-9]*$/) || number.length > 8;
 }
 
+export const scrollToFirstError = (formErrors) => {
```

Comment:
(I have very little FE knowledge) would something like this be useful https://www.npmjs.com/package/react-scroll-into-view

Reply:
Oh wow, neat feature. 

I think we can keep this method for now. But that's a cool package !